web/router.ex

```diff
@@ -75,6 +75,7 @@ defmodule CoursePlanner.Router do
     end
 
     resources "/settings", SettingController, only: [:show, :edit, :update], singleton: true
+    resources "/aboutclientprogram", AboutClientProgramController, only: [:show], singleton: true
```

Comment:
at this point one instance of course planner is for one program, so if Dina has a program it will have a normal about for that instance